front/vite.config.ts

```diff
@@ -38,6 +38,7 @@ export default defineConfig(({ mode }) => {
     server: {
       port: 3000,
       host: '0.0.0.0',
+      allowedHosts: true,
```

Comment:
Needed this to be able to test CLaveUnica locally via local tunnel

Reply:
What does it do?